tests/kafkatest/services/streams.py

```diff
@@ -50,6 +50,7 @@ def __init__(self, test_context, kafka, streams_class_name, user_test_args):
         self.kafka = kafka
         self.args = {'streams_class_name': streams_class_name,
                      'user_test_args': user_test_args}
+        self.log_level = "DEBUG"
```

Comment:
Yeah, it's a good time to turn on logging in 0.10.2 to help debug. So this is good.